content/scalability/docs/supporting-infrastructure.md

```diff
@@ -0,0 +1,228 @@
+
+Global Accelerator can route workloads to the nearest region based on request IP address. This may be useful for workloads that are deployed to multiple regions, but it does not improve routing to a single cluster in a single region. Using Route 53 in combination with the Global Accelerator has additional benefits such as health checking and automatic failover if an AZ is not available. You can see an example of using Global Accelerator with Route 53 in [this blog post](https://aws.amazon.com/blogs/containers/operating-a-multi-regional-stateless-application-using-amazon-eks/).
+
+CloudFront can be use with Route 53 and Global Accelerator or by itself to route traffic to multiple destinations. CloudFront caches assets being served from the origin sources which may reduce bandwidth requirements depending on what you are serving.
+
+## Use AWS Systems Manager Patch Manager to automate patching nodes
+
+To patch a Linux host it takes seconds to install a package which can be run in parallel without disrupting containerized workloads. The package can be installed and validated without cordoning, draining, or replacing the instance.
+
+To replace an instance you first need to create, validate, and distribute new AMIs. The instance needs to have a replacement created, and the old instance needs to be cordoned and drained. Then workloads need to be created on the new instance, verified, and repeated for all instances that need to be patched. It takes hours, days, or weeks to replace all instances in a large cluster safely without disrupting workloads.
+
+Amazon recommends using immutable infrastructure that is built, tested, and promoted from an automated, declarative system, but if you have a requirement to patch systems quickly then you will likely need to patch existing systems in place. Because of the large time differential between patching and replacing systems we recommend using [AWS Systems Manager Patch Manager](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-patch.html) to automate patching nodes when required to do so.
+
+Patching nodes will allow you to quickly roll out security updates and replace the instances on a regular schedule after your base AMI has been updated and promoted. If you are using Bottlerocket we recommend using the [Bottlerocket update operator](https://github.com/bottlerocket-os/bottlerocket-update-operator) to keep your nodes up to date.
```

Comment:
How does this square with Karpenter? Since you're mentioning Karpenter a lot in this section, you might want to consider adding a paragraph on that.

Reply:
I'm not sure I understand. You can still patch karpenter nodes with patch manager and have them automatically replaced with TTL.

Reply:
I meant updating BR. I guess you could still use SSM to do that OR you could use their update operator. How do you think customers should update BR when it's used with Karpenter? @rothgar

Reply:
BR is a read only FS and has no package manager so it cannot be patched in place. You can either replace the instance with new AMIs or us the update operator which will install the update on a fresh partition and reboot the instance. I'll add clarification for that.